### ISSUE TITLE
Allow task assignment from new task page

### ISSUE THREAD

**Issue by begedin** [code-corps/code-corps-ember#1174]
# Problem

Users which are allowed to do so should be able to assign tasks from the new task page. We should use the component extracted in #1172 to achieve this, so this will be marked as blocked until we're ready for that.

Since this is for a new task, it should be implemented similarly to skill assignments. Instead of immediately creating a `userTask` record as soon as the user makes a dropdown selection, we should assign the selected user to a controller/route property. Once the user then saves the task, we should create and save the `userTask` record automatically.

This means that the component defined in #1172 might have to be modified to allow external assignment of the user, and tests would have to be updated to support this.

## Subtasks

- [ ] Include component from #1172 in `project/tasks/task/new.hbs`
- [ ] Modify component so it allows external assignment of the assigned user. This should be relatively simple and be just a matter of overwriting the component property.
- [ ] Update component tests so they cover this case
- [ ] Handle `onChange` action correctly in route/controller. It should work similarly to how skill assignment works.
- [ ] Modify task save action on route/controller to also create a `userTask` once saving is done
- [ ] Fix any styling issues that might occur from using the component in the sidebar instead of the task card.
